src/shared/components/oncoprint/ResultsViewOncoprintUtils.ts

```diff
@@ -52,11 +56,26 @@ export function makeProfiledInClinicalAttributes(
                 }
             }
         });
+
```

Comment:
i think this whole thing needs testing.  i mean, if anything does, it's this, right?

Reply:
good point, I covered it with a screen shot but Ill add a case for this to the spec file